sensor_msgs/JointState js
geometry_msgs/Wrench eef_force
geometry_msgs/Pose eef_target
---
std_msgs/Bool is_valid
sensor_msgs/JointState je_diff